docs/document/website/versioned_docs/version-1.6.0/components/mobile/mobile-date-picker.md

```diff
@@ -0,0 +1,90 @@
+---
+id: version-1.6.0-mobile-date-picker
+title: MobileDatePicker
+sidebar_label: MobileDatePicker
+original_id: mobile-date-picker
+---
+
+## Overview
```

Comment:
Please create the same file with Japanese language at docs/document/website/translated_docs/ja/version-1.6.0